lib/ui/widgets/profile_password.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+
+class PasswordField extends StatefulWidget {
+  final String labelText;
+  final String? hintText;
+  final bool includeTextField;
+
+  const PasswordField({
+    super.key,
+    required this.labelText,
+    this.hintText,
+    this.includeTextField = true,
+  });
+
+  @override
+  PasswordFieldState createState() => PasswordFieldState();
+}
+
+class PasswordFieldState extends State<PasswordField> {
```

Comment:
**suggestion:** I think we should merge this logic into the custom one. They share 99% of tf code and giving the developer the option to make any texfield obscured is a nice add on :)

Reply:
done